modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -8,21 +8,68 @@ This guide contains an overview of the changes in Canvas Kit v9. Please
 [reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you have
 any questions.
 
+- [Glossary](#glossary)
+  - [Main](#main)
+  - [Preview](#preview)
+  - [Labs](#labs)
 - [Codemod](#codemod)
-- [Component Deprecations](#component-deprecations)
+- [Deprecations and Removals](#deprecations-and-removals)
```

Comment:
Ah nvm I see. Should we add to the glossary deprecation vs removal?

Reply:
Sure. Another option would be to include the definitions of deprecation and removal in the `Deprecations and Removals` section since those terms are most likely to appear in that section (as opposed to throughout the entire document as is the case for the package names in the Glossary).

I could go either way on this. I say we try it both ways and see how it looks.